src/core_plugins/kibana/public/dashboard/dashboard.html

```diff
@@ -34,20 +34,20 @@
               kbn-typeahead-input
               ng-model="model.query"
               placeholder="Filter..."
-              aria-label="Filter input"
+              aria-label="Enter query"
```

Comment:
Query probably makes more sense to end users since they're using the lucene query syntax. If we're changing the aria-label we should probably change the placeholder text as well right?